src/ontology/mondo-edit.obo

```diff
@@ -125013,31 +124996,41 @@ id: MONDO:0006702
 name: chronic inflammatory demyelinating polyradiculoneuropathy
 def: "A rare neurological disorder in which there is inflammation of nerve roots and peripheral nerves and destruction of the fatty protective covering (myelin sheath) over the nerves. This affects how fast the nerve signals are transmitted and leads to loss of nerve fibers. This causes weakness, paralysis and/or impairment in motor function, especially of the arms and legs (limbs). Sensory disturbance may also be present. The motor and sensory impairments usually affect both sides of the body (symmetrical), and the degree of severity and the course of disease may vary from case to case. Some affected individuals may follow a slow steady pattern of symptoms while others may have symptoms that stabilize and then relapse." [https://rarediseases.org/rare-diseases/chronic-inflammatory-demyelinating-polyneuropathy/]
 subset: gard_rare {source="GARD:6102"}
+subset: inferred_rare
```

Comment:
@nicolevasilevsky should this `subset` be added? In a previous [ "split" PR](https://github.com/monarch-initiative/mondo/pull/7320/files#r1501269691) Sabrina said subset tags should not be added.

Reply:
@sabrinatoro these rare tags came from the obsoleted/merged term- should I remove them or it is okay to move them from the obsolete term to the merged term in cases of a merge?

Reply:
@nicolevasilevsky did you hear back from Sabrina or do you want to merge this one as-is for now and we'll sort out the policy on subset at next week's curation call?

Reply:
I haven't heard back, let's merge before we get conflicts and we can revise later if needed. Good idea, thanks!

Reply:
Sounds good!